service/src/main/java/se/inera/statistics/service/warehouse/IntygType.java

```diff
@@ -46,15 +46,18 @@ public enum IntygType {
     TSTRK1062("TSTRK1062", "tstrk1062", "TSTRK1062", "TSTRK1062 Transportstyrelsens läkarintyg ADHD", true),
     AG114("AG1-14", "ag114", "AG1-14", "AG1-14 Läkarintyg om arbetsförmåga - sjuklöneperioden", true),
     AG7804("AG7804", "ag7804", "AG7804", "AG7804 Läkarintyg om arbetsförmåga - arbetsgivare", true),
-    FK7211("IGRAV", "fk7211", "FK 7211", "Intyg om graviditet", true),
-    FK7443("ITFP", "fk7443", "FK 7443", "Intyg om tillfällig föräldrapenning", true);
+    FK7210("IGRAV", "fk7210", "FK 7210", "Intyg om graviditet", true),
+    FK7472("ITFP", "fk7472", "FK 7472", "Intyg om tillfällig föräldrapenning", true),
+    FK3226("LUNSP", "fk3226", "FK 3226", "Läkarutlåtande för närståendepenning", true),
+    FK7809("LUMEK", "fk7809", "FK 7809", "Läkarutlåtande för merkostnadsersättning", true);
 
     private static final ImmutableSet<IntygType> INCLUDED_IN_KOMPLETTERING_REPORT = ImmutableSet.of(LISJP, LUSE, LUAE_NA, LUAE_FS);
 
     private static final ImmutableSet<IntygType> IS_SJUKPENNING = ImmutableSet.of(LISJP, FK7263);
 
     private static final ImmutableList<IntygType> INCLUDED_IN_INTYG_TOTALT_REPORT = ImmutableList.of(
-        AF00213, AF00251, AG114, AG7804, DB, DOI, LISJP, LUSE, LUAE_NA, LUAE_FS, TSTRK1009, TSTRK1007, TSTRK1031, TSTRK1062, FK7211);
+        AF00213, AF00251, AG114, AG7804, DB, DOI, LISJP, LUSE, LUAE_NA, LUAE_FS, TSTRK1009, TSTRK1007, TSTRK1031, TSTRK1062, FK7210,
+        FK7472, FK3226, FK7809);
 
     private static final ImmutableList<IntygType> INCLUDED_IN_INTYG_FILTER = ImmutableList.of(SJUKPENNING, LUSE, LUAE_NA, LUAE_FS);
```

Comment:
Do you know what this "INCLUDED_IN_INTYG_FILTER" is used for? It seems to be all FK certificates. Could the new types be relevant here?

Reply:
Isn't this a list of types that should be included in the statistics for sickleave?

Reply:
Is it? I would imagine that it should only be SJUKPENNING, because the other types are not part of sick leaves as far as I know.

Reply:
Looks like this is used in the filter for the administrative contact/messages pages so I added FK3226 and FK7809 as they will have it. FK7210 will have no communication at all and FK7442 only allows completion and no administrative contact.

Reply:
Nice! Sounds good!